aip-console-tools-cli/README.md

```diff
@@ -131,10 +131,18 @@ to create a new Version.
 To **Onboard Application** using a **file full path**, the process will upload sources on the CAST Imaging server. The
 file should be
 accessible from the machine where the batch is executed.
-To perform that action you can inspire from the following command:
+
+The command is using following strategies:
+
+- *FAST_SCAN*: for the sources' delivery contents without running the analysis. This strategy can be used as much as
+  required by providing the sources file.
+- *DEEP_SCAN*: to trigger the analysis after the FIRST_SCAN has been performed. This also manage to upload data to CAST
+  imaging depending on the available configuration.
+
+To perform that action you can inspire from the following command (see advance usage for more details):
 
 ```bash
-java -jar .\aip-console-tools-cli.jar Onboard-Application --apikey="valid.key" -n "my app" --domain-name="Your Domain" -f "C:\folder\some-location\sources-file.zip" --verbose=false --exclude-patterns="tmp/, temp/, *test, tests, target/, .svn/, .git/, _Macosx/, test/"
+java -jar .\aip-console-tools-cli.jar Onboard-Application --apikey="valid.key" -n "my app" --domain-name="Your Domain" -f "C:\folder\some-location\sources-file.zip" --strategy=FAST_SCAN --verbose=false --exclude-patterns="tmp/, temp/, *test, tests, target/, .svn/, .git/, _Macosx/, test/"
```

Comment:
what is -strategy  remove it,  the name of the CLI shuld be fast-scan instead of onboarding

Reply:
got it